addons/mail/models/mail_alias.py

```diff
@@ -228,22 +255,43 @@ def _get_alias_bounced_body_fallback(self, message_dict):
         contact_description = self._get_alias_contact_description()
         default_email = self.env.company.partner_id.email_formatted if self.env.company.partner_id.email else self.env.company.name
         return Markup(
-            _("""<p>Dear Sender,<br /><br />
-The message below could not be accepted by the address %(alias_display_name)s.
+            '<p>' +
```

Comment:
Maybe we could keep some of the "br" to make a complete message to translate, something like
```
    content = Markup(
        _("""The message below could not be accepted by the address %(alias_display_name)s.
          Only %(contact_description)s are allowed to contact it.<br /><br />
          Please make sure you are using the correct address or contact us at %(default_email)s instead."""
        )
    ) % {
        'alias_display_name': self.display_name,
        'contact_description': contact_description,
        'default_email': default_email,
    }
    return Markup('<p>%(header)s<br /><br />%(content)s<br /><br />%(regards)s</p>') % {
        'content': content,
        'header': _('Dear Sender'),
        'regards': _('Kind Regards'),
    }
```

Reply:
Yes indeed, that’s simpler for the translate team, thanks.